charts/fastly-controller/templates/_helpers.tpl

```diff
@@ -0,0 +1,59 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "fastly-controller.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```

Comment:
I think this should be `{{- end }}` because chomping the trailing whitespace can cause subtle template bugs.